src/pages/objkt-display/index.js

```diff
@@ -19,10 +20,10 @@ import { Page, Container, Padding } from '@components/layout'
 import { renderMediaType } from '@components/media-types'
 import { ItemInfo } from '@components/item-info'
 import { Menu } from '@components/menu'
-import { Info, Collectors, Swap, Burn, History } from './tabs'
+import { Info, Collectors, Swap, Burn, History, Transfer } from './tabs'
 import styles from './styles.module.scss'
 import './style.css'
-import { Transfer } from '@components/collab/show/Transfer'
+import useSettings from 'hooks/use-settings'
```

Comment:
`hooks/use-settings` => `@hooks/use-settings`

Reply:
Yep, I have a few of them, that's vscode auto import, strangely it works 😅